requirements/base.txt

```diff
@@ -6,4 +6,5 @@ tensorboard>=1.14
 future>=0.17.1  # required for builtins in setup.py
 # pyyaml>=3.13
 PyYAML>=5.1  # OmegaConf requirement
-tqdm>=4.41.0
\ No newline at end of file
+tqdm>=4.41.0
+packaging
```

Comment:
it is native, no need to add?
```suggestion
```

Reply:
I tried without but it failed.
How do I check the version natively?

Reply:
@Borda

Reply:
Then keep it for now...

Reply:
or use distrib utils - https://stackoverflow.com/a/46427037/4521646